docs/data/base/components/badge/badge.md

```diff
@@ -8,45 +8,76 @@ packageName: '@mui/base'
 
 # Unstyled badge
 
-<p class="description">The `BadgeUnstyled` component generates a small label that is attached to its children elements.</p>
+<p class="description">The BadgeUnstyled component generates a small label that is attached to its child element.</p>
 
-```js
+## Introduction
+
+A badge is a small descriptor for UI elements.
+It typically sits on or near an element and indicates the status of that element by displaying a number, icon, or other short set of characters.
+
+The `BadgeUnstyled` component creates a badge that is applied to its child element.
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+## Component
+
+### Anatomy
+
+After [installation](/base/getting-started/installation/), you can start building with this component using the following basic elements:
+
+```jsx
 import BadgeUnstyled from '@mui/base/BadgeUnstyled';
 ```
 
-{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+```jsx
+<BadgeUnstyled>{/* the element that the badge is attached to */}</BadgeUnstyled>
```

Comment:
Would a single code block rather than two improve the UX? I feel that it would for me:

<img width="782" alt="Screenshot 2022-06-15 at 15 24 06" src="https://user-images.githubusercontent.com/3165635/173837950-1a1eeeef-e9d0-4274-8d15-93323fafbe95.png">

Reply:
I had these as a single code block originally, but Prettier was doing some odd things to them—mainly adding semicolons like this:

```jsx
<BadgeUnstyled>{/* the element that the badge is attached to */}</BadgeUnstyled>;
```

The only way I could figure out to stop this was to split them into two sections.

Reply:
@oliviertassinari I don't think one code block would improve DX. Two blocks indicate these lines are separate and should not be pasted together. Imports will go to the top of the code file, while the component goes below and must be prepended with something (a `return` in a function, variable declaration, etc).

Reply:
@mnajdova and I discussed this today and agreed that @danilo-leal's pattern above is ideal, so I'm going to run with it in the Base docs!

Additionally, w/r/t the info that's specific to the components (vs. hooks), we decided that we should document the interior slots more explicitly, as I did on the Unstyled Switch page:

<img width="855" alt="Screen Shot 2022-06-21 at 9 00 12 AM" src="https://user-images.githubusercontent.com/71297412/174818877-f5c9c10e-bb13-49f5-92d9-b7ddefdec752.png">

cc @michaldudak